README.md

```diff
@@ -26,9 +26,12 @@ A collection of our Android templates:
   ```   
     package-name=                      New package name (i.e., com.example.package)
     app-name=                          New app name (i.e., MyApp, "My App", "my-app")
+    template=                          Template type (i.e., xml, compose)
   ```
 
-  Example: `kscript new_project.kts package-name=co.myproject.example app-name="My Project"`
```

Comment:
Since you already updated `README.md` would you mind updating `Home.md` too?

If you agree, please include the issue number (https://github.com/nimblehq/android-templates/issues/338) in your pull request & commit history